docs/section4/4-6.md

```diff
@@ -0,0 +1,37 @@
+# ASHVAC – Rule 4-6
+
+**Rule ID:** 4-6
+ 
+**Rule Description:** HVAC fans shall remain on during occupied and unoccupied hours in systems primarily serving computer rooms.  
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-4 exception #3 Schedules for the proposed building  
+**Appendix G Section Reference:**  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:** 
+
+1. P-PRM is modeled with a least one computer room space. 
+
+**Manual Check:** None  
+**Evaluation Context:** Evaluate each data element  
+**Data Lookup:** None  
+**Function Call:** None
+
+## Rule Logic:
+- For each zone in P_RMR: `for zone_p in P_RMR...zones:`
+    - For each space in zone: `for space_p in zone_p.zones:`
+        - Check if space(s) is of type that has health and safety mandated minimum ventilation requirements during unoccupied hours: `if space_p.lighting_space_type in [COMPUTER_ROOM]:`
```

Comment:
Please update to "Check if space(s) is of type computer_room...."

Reply:
Done